### PR TITLE
Installation of Chakra, Login Page, Loading Page, revision of past home page and navbar.

### PR DESCRIPTION
**Pull request by DiegoLandaeta03**
- After I installed Chakra, I had to make adjustments to previous components to make them look like they did before. The main thing was switching from using `<div>` to `<Box>` which is Chakra's version of a div [[ref](https://v2.chakra-ui.com/docs/components/box)]. 
- Created  a login page with Chakra components and matched styling of profile button on home page.
- Created a loading state page in between Spotify login and home page.

Shows log in process:
Logging in

Shows loading state:
LoadingState

### CODE REVIEW

**Comment by roeshanigos** [comment]
nit: can you make the first screenrecording bigger? it's very pixelated right now even when i zoom in!

File: front-end/src/SpotifyCallback.jsx [hunk 1]
```diff
@@ -65,14 +67,31 @@ function SpotifyCallback() {
                 })
                 .catch(error => {
                     console.error('Error:', error);
+                })
+                .finally(() => {
+                    setTimeout(() => {
+                        setIsLoading(false);
+                    }, 3000);
```
Comment (roeshanigos) [inline front-end/src/SpotifyCallback.jsx:74]: any reasoning behind 3000?

File: front-end/src/Home.jsx [hunk 2]
```diff
@@ -9,13 +10,13 @@ function Home() {
     return (
         <div className='Home'>
             <header>
-                <Navbar username={username}/>
+                <Navbar username={username} />
             </header>
             <main>
                 <div className='beatTitle'>
-                    <h2>Your Beats</h2>
+                    <Heading as='h2' size='2xl'>Your Beats</Heading>
```
Comment (roeshanigos) [inline front-end/src/Home.jsx:17]: Do you want to call each of these Beats or Stitches (from the app title)

**Comment by roeshanigos** [comment]
Last thing - is it possible for Spotify login to not modal (like a floating card) instead of a separate page?

File: front-end/src/Home.jsx (hunk 2 above)
Comment (DiegoLandaeta03) [reply front-end/src/Home.jsx:17]: Ah yes, I will switch it! Love the idea thank you

File: front-end/src/SpotifyCallback.jsx (hunk 1 above)
Comment (DiegoLandaeta03) [reply front-end/src/SpotifyCallback.jsx:74]: > any reasoning behind 3000?
Its not too fast or too slow, I tested out different times and decided on 3000.

**Comment by DiegoLandaeta03** [comment]
> Last thing - is it possible for Spotify login to not modal (like a floating card) instead of a separate page?

Sadly that's not possible since the way Spotify makes you log in is through their website.

**Comment by DiegoLandaeta03** [comment]
> nit: can you make the first screenrecording bigger? it's very pixelated right now even when i zoom in!

The GIF even with high quality wasn't too good so I put the actual video I took, hope that's better!

**Comment by alegleason** [comment]
Thanks for the screen recording! Love the GIF but I understand the quality issue 😝

**Comment by alegleason** [comment]
> The main thing was switching from using ? to which is Chakra's version of a div [[ref](https://v2.chakra-ui.com/docs/components/box)].

Did you miss a reference on the summary?

**Comment by DiegoLandaeta03** [comment]
> > The main thing was switching from using ? to which is Chakra's version of a div [[ref](https://v2.chakra-ui.com/docs/components/box)].
> 
> Did you miss a reference on the summary?

Oh yeah, I didn't notice but since it was code it wasn't showing up. Updated it with the ``, thank you!

File: front-end/src/Navbar.jsx [hunk 3]
```diff
@@ -42,28 +43,40 @@ function Navbar({ username }) {
...
+                        <Text fontSize={'lg'}>Menu</Text>
+                    </button>
                     {isDropdownVisible && (
-                        <div className="dropdownMenu">
+                        <div className="dropdownMenu" style={{ position: 'absolute', right: 0 }}>
                             <ul>
-                                <li onClick={handleHome}>Home</li>
-                                <li onClick={handleCreate}>Create</li>
-                                <li onClick={handleLogout}>Logout</li>
+                                <li onClick={handleHome}>
```
Comment (alegleason) [inline front-end/src/Navbar.jsx:65]: I'd probably advice more descriptive names here. `handleHome` doesn't tell me much, while `didClickHomeButton`, or `didClickHomeItem` makes more sense.

File: front-end/src/SpotifyLogin.jsx [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+                    bg="linear-gradient(0deg, rgba(115, 41, 123, 0.9) 10%, rgb(81, 40, 138, 0.9) 100%)"
+                    size="lg"
+                    as="a"
+                    href={`${import.meta.env.VITE_BACKEND_ADDRESS}/auth/login`}
+                    color={"white"}
+                    transition="background 0.3s ease-in-out"
+                    _hover={{ color: "black", 
+                    bg: "white",
+                    transition: "background 0.3s ease-in-out",
+                    boxShadow: "0 0 20px -2px rgba(195, 111, 199, .5)" }}>
```
Comment (alegleason) [inline front-end/src/SpotifyLogin.jsx:21]: Would it be best to have this on a CSS file?

File: front-end/src/Navbar.jsx (hunk 3 above)
Comment (DiegoLandaeta03) [reply front-end/src/Navbar.jsx:65]: Got it, will rename in the next commit.

File: front-end/src/SpotifyLogin.jsx (hunk 4 above)
Comment (DiegoLandaeta03) [reply front-end/src/SpotifyLogin.jsx:21]: This is how Chakra does their css, there are some things that I am doing in CSS. But I am using the tags that I find from Chakra documentation and examples.

**Pull request merged**